exercises/Modules-1-R.md

```diff
@@ -0,0 +1,17 @@
+—-—
+layout: exercise
+title: Modules 1
+language: R
+---
+
+Use the `sqrt()` and `log()` functions from the `math` module, along with the
```

Comment:
Done.

I chose not to make any changes to this file originally because using
built-in functions for the problem negates its utility as a 'Module'
exercise. We'll have to redesign the problem to make it use a package.

On Wed, Apr 15, 2015 at 4:30 PM, Ethan White <notifications@github.com>
wrote:

> In exercises/Modules-1-R.md
> <https://github.com/ethanwhite/datacarp-semester-biology/pull/13#discussion_r28465017>
> :
>
> > @@ -0,0 +1,17 @@
> > +—-—
> > +layout: exercise
> > +title: Modules 1
> > +language: R
> > +---
> > +
> > +Use the `sqrt()` and `log()` functions from the `math` module, along with the
>
> These are built-ins in R.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ethanwhite/datacarp-semester-biology/pull/13/files#r28465017>
> .
>